### PR TITLE
Adds Nightvision Chameleon Glasses

### PR DESCRIPTION
**Pull request by HMBGERDO**
## What Does This PR Do

This PR adds nightvision chameleon glasses, similar to nightvision googles but chameleon, to traitor uplink for 5 TC

## Why It's Good For The Game

While we have nightvision module with cost of 5 TC for a modsuit, we have nothing for people who dont wear modsuit. You can buy thermals for 15 TC but like, sometimes you dont have 15 TC and thermal doesnt provide as good nightvision as actual nightvision googles.

## Testing

Compiled, checked traitor uplink, bought glasses.

## Changelog
:cl:
add: Nightvision Chameleon Glasses is now available in all uplinks with cost of 5 TC. Its same as nightvision googles, but chameleon. Watch your eye damage, agents.
/:cl:

### CODE REVIEW

File: code/datums/uplink_items/uplink_general.dm [hunk 1]
```diff
@@ -503,6 +503,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/clothing/glasses/chameleon/thermal
 	cost = 15
 
+/datum/uplink_item/stealthy_tools/thermal
```
Comment (1080pCat) [inline code/datums/uplink_items/uplink_general.dm]: why is this under thermals? 
should really be pathed `...stealthy_tools/nightvision`

Comment (Spaghetti-bit) [inline code/datums/uplink_items/uplink_general.dm:506]:
```suggestion
/datum/uplink_item/stealthy_tools/thermal/chamnightvision
```

Your new datum had the exact same path as regular thermals. I don't think these new glasses are thermals ^v^

**Review by Spaghetti-bit — CHANGES_REQUESTED** [review]
code looks good.

File: code/datums/uplink_items/uplink_general.dm (hunk 1 above)
Comment (Spaghetti-bit) [reply code/datums/uplink_items/uplink_general.dm]: Good to resolve, outdated. @HMBGERDO

**Review by 1080pCat — APPROVED** [review]

**Review by Sirryan2002 — APPROVED** [review]

**Review by DGamerL — APPROVED** [review]

**Pull request merged**